packages/devtools_app/test/shared/history_manager_test.dart

```diff
@@ -81,5 +81,57 @@ void main() {
       history.moveBack();
       expect(history.current.value, ref1);
     });
+
+    test('replaceCurrent empty history', () {
+      expect(history.current.value, null);
+      history.replaceCurrent(ref1);
+      expect(history.current.value, ref1);
+      expect(history.hasNext, false);
+      expect(history.hasPrevious, false);
+    });
+
+    test('replaceCurrent at the top of the stack', () {
+      history.push(ref1);
+      history.push(ref2);
+      history.push(ref3);
```

Comment:
after this line, verify that the current value is ref3. That way, we know for sure that the replaceCurrent call changed the value